Cargo.toml

```diff
@@ -14,8 +14,10 @@ license = "MIT"
 pkg-config = "*"
 
 [dependencies]
+libc = "*"
 log = "*"
 time = "*"
+thread-scoped = "*"
```

Comment:
Yeah, probably so.  I guess "x.y" is appropriate? (leaving the third .z unspecified)